### ISSUE TITLE
Can't rotate LCS

### ISSUE THREAD

**Issue by scrouthtv** [Zolko-123/FreeCAD_Assembly4#296]
Hello,

after upgrading from v0.11.5 to v0.11.9, I can't rotate LCS anymore:
![rotate](https://user-images.githubusercontent.com/23227405/154493226-e74ea481-9554-4910-8a82-bfb8ad64b50d.gif)

Am I doing something wrong?

**Comment by Zolko-123** [comment]
> Am I doing something wrong?

yes : if you attach an LCS to a point, it doesn't have any orientation references. See

https://wiki.freecadweb.org/Part_EditAttachment
